docs/data/base/components/click-away-listener/click-away-listener.md

```diff
@@ -1,58 +1,59 @@
 ---
 product: base
-title: Detect click outside React component
+title: React ClickAwayListener component
 components: ClickAwayListener
 githubLabel: 'component: ClickAwayListener'
 packageName: '@mui/base'
 ---
 
 # Click-away listener
 
-<p class="description">Detect if a click event happened outside of an element. It listens for clicks that occur somewhere in the document.</p>
+<p class="description">The <code>ClickAwayListener</code> component detects when a click event happens outside of its children elements.</p>
 
-- 📦 [981 B gzipped](/size-snapshot).
+- 📦 [981 B gzipped](/size-snapshot)
 - ⚛️ Supports portals
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
-## Example
+## Basic usage
 
-For instance, if you need to hide a menu dropdown when people click anywhere else on your page:
+The following demo shows how to hide a menu dropdown when users click anywhere else on the page:
 
 {{"demo": "ClickAway.js"}}
 
-Notice that the component only accepts one child element.
+> **Note:** the `ClickAwayListener` component only accepts _one_ child element.
+
 You can find a more advanced demo on the [Menu documentation section](/components/menus/#menulist-composition).
 
 ## Portal
 
-The following demo uses [`Portal`](/base/react-portal/) to render the dropdown into a new "subtree" outside of current DOM hierarchy.
+The following demo uses the [`Portal`](/base/react-portal/) component to render the dropdown into a new subtree outside of the current DOM hierarchy:
 
 {{"demo": "PortalClickAway.js"}}
 
 ## Leading edge
 
-By default, the component responds to the trailing events (click + touch end).
-However, you can configure it to respond to the leading events (mouse down + touch start).
+By default, the `ClickAwayListener` component responds to **trailing events**—the _end_ of a click or touch.
+
+You can set the component to listen for **leading events** (the start of a click or touch) using the `mouseEvent` and `touchEvent` props, as shown in the following demo:
 
 {{"demo": "LeadingClickAway.js"}}
 
-> ⚠️ In this mode, only interactions on the scrollbar of the document is ignored.
+> ⚠️ **Note:** When the component is set to listen for leading events, interactions with the scrollbar are ignored.
 
 ## Accessibility
 
-By default `<ClickAwayListener />` will add an `onClick` handler to its children.
-This can result in e.g. screen readers announcing the children as clickable.
-However, the purpose of the `onClick` handler is not to make `children` interactive.
+By default, `ClickAwayListener` will add an `onClick` handler to its children.
+This can result in screen readers announcing that the children are clickable, even though this `onClick` handler has no effect on the childen elements themselves.
```

Comment:
```suggestion
This can result in screen readers announcing that the children are clickable, even though this `onClick` handler has no effect on the children elements themselves.
```